### PR TITLE
feat: add login and signup page

### PR DESCRIPTION
**Pull request by nerdyspook**
> added login and signup page
> fixed content layout of wishlist and products page

### CODE REVIEW

File: src/pages/Auth/Auth.scss [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+        }
+    }
+}
+
+@media screen and (min-width: 768px) {
+    .auth_container {
+        display: grid;
+        grid-template-columns: 1fr;
+        margin-top: 4rem;
+        // background-color: red;
```
Comment (prathmeshjagtap) [inline src/pages/Auth/Auth.scss]: you can remove commneted
 code

File: src/pages/Auth/Auth.scss [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+        .split-screen {
+            flex-direction: row;
+            height: calc(100vh - 4rem);
+            // margin-top: var(--header-height);
+            // margin-top: 3rem;
+            // max-height: calc(100vh - var(--header-height));
+            // height: calc(100vh - var(--header-height));
+            // height: calc(100vh - var(--header-height) + 0.7rem);
```
Comment (prathmeshjagtap) [inline src/pages/Auth/Auth.scss]: you can remove this if not needed

**Review by prathmeshjagtap — APPROVED** [review]
Everything else is good, the code is clean and well structured just remove the commented code.

File: src/pages/Auth/Auth.scss (hunk 2 above)
Comment (nerdyspook) [reply src/pages/Auth/Auth.scss]: Thanks, I missed this part while removing commented code

**Review by parthgabani777 — APPROVED** [review]

**Pull request merged**